lib/fig/repository.rb

```diff
@@ -466,9 +469,10 @@ def fig_file_for_package_download(package_download_dir)
     File.join(package_download_dir, PACKAGE_FILE_IN_REPO)
   end
 
-  def remote_directory_for_package(descriptor)
+  def remote_directory_for_package(descriptor, for_publishing = false)
```

Comment:
If you make this a keyword argument instead of defaulting it to `false`, it will be `nil` instead of `false` but will still be falsey in the ternary. But it means you don't need to put comments at all the call sites explaining what the bare true means.
```suggestion
  def remote_directory_for_package(descriptor, for_publishing:)
```

Reply:
Integrated in 36c6405